src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -2199,17 +2199,16 @@ void MacroAssembler::cmpxchgptr(Register oldv, Register newv, Register addr, Reg
   // addr identifies memory word to compare against/update
   Label retry_load, nope;
   bind(retry_load);
-  // flush and load exclusive from the memory location
```

Comment:
Could you please also fix the comments in MacroAssembler::atomic_incw? Thanks

Reply:
atomic_incw has not been called anywhere, so I'll remove it in the next commit.